packages/taco/test/taco-sign.test.ts

```diff
@@ -1,79 +1,395 @@
 import {
-  convertUserOperationToPython,
+  EncryptedThresholdSignatureRequest,
+  PackedUserOperation,
+  SessionStaticKey,
+  SessionStaticSecret,
+  SignatureResponse,
+  UserOperation,
+} from '@nucypher/nucypher-core';
+import {
+  fromHexString,
+  initialize,
+  isPackedUserOperation,
+  PackedUserOperationToSign,
   PorterClient,
+  SignerInfo,
   SigningCoordinatorAgent,
+  toCorePackedUserOperation,
+  toCoreUserOperation,
+  UserOperationToSign,
 } from '@nucypher/shared';
 import { fakePorterUri } from '@nucypher/test-utils';
 import { ethers } from 'ethers';
-import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { beforeAll, beforeEach, describe, expect, it, vi } from 'vitest';
 
 import { ContractCondition } from '../src/conditions/base/contract';
 import { RpcCondition } from '../src/conditions/base/rpc';
 import { CompoundCondition } from '../src/conditions/compound-condition';
 import { ConditionExpression } from '../src/conditions/condition-expr';
 import { setSigningCohortConditions, signUserOp } from '../src/sign';
 
+import { mockMakeSessionKey } from './test-utils';
+
+function getNumberValue(value: bigint | number): bigint {
```

Comment:
Nitpick: not sure if this function name is clear. What about

```suggestion
function toBigInt(value: bigint | number): bigint {
```

Reply:
✔️